src/protocolsupport/protocol/transformer/mcpe/handler/ClientboundPacketHandler.java

```diff
@@ -92,7 +93,11 @@
 import net.minecraft.server.v1_8_R3.World;
 
 public class ClientboundPacketHandler {
-
+	private static final RemappingTable blockRemapper = IdRemapper.BLOCK.getTable(ProtocolVersion.MINECRAFT_PE);
```

Comment:
All of them should be protected, or they will be copied to every anonymous class in transformers registry